src/utils/richtext.tsx

```diff
@@ -1,6 +1,6 @@
-import { PortableTextReactComponents } from "@portabletext/react";
+import { PortableTextReactResolvers } from "@kontent-ai/rich-text-resolver/utils/react";
 
-export const defaultPortableRichTextComponents: Partial<PortableTextReactComponents> = {
+export const defaultPortableRichTextResolvers: PortableTextReactResolvers = {
   list: {
     bullet: ({ children }) => <ul className="text-xl text-gray-700 list-disc ml-8">{children}</ul>,
     number: ({ children }) => <ol className="text-xl text-gray-700 list-decimal ml-8">{children}</ol>,
```

Comment:
you can get rid of `marks` altogether, if you use `<PortableText>` from utils/react of the resolver, rather than the external portable text package, as it defines defaults for `sub` and `sup`